util/topgen/templates/toplevel_racl_pkg.sv.tpl

```diff
@@ -8,6 +8,12 @@ package top_racl_pkg;
   // Number of RACL policies used
   parameter int unsigned NrRaclPolicies = ${racl_config['nr_policies']};
 
+  // RACL Policy selector bits
+  parameter int unsigned RaclPolicySelLen = $clog2(NrRaclPolicies);
```

Comment:
```suggestion
  parameter int unsigned RaclPolicySelLen = prim_util_pkg::vbits(NrRaclPolicies);
```
Because `$clog2(1) == 0`, hence the vector below would be `[-1:0]`; but we want `[0:0]`, which `vbits(1) == 1` provides.